source/lib/rocprof-sys/library/amd_smi.cpp

```diff
@@ -350,16 +392,18 @@ data::post_process(uint32_t _dev_id)
                 auto addendum = [&](const char* _v) {
                     return JOIN(" ", "GPU", _v, JOIN("", '[', _dev_id, ']'), "(S)");
                 };
-                auto addendum_blk = [&](std::size_t _i, const char* _metric) {
-                    if(_i < 10)
+                auto addendum_blk = [&](std::size_t _i, const char* _metric,
+                                        bool is_busy = false, std::size_t xcp_idx = 0) {
+                    if(is_busy)
                     {
                         return JOIN(" ", "GPU", JOIN("", '[', _dev_id, ']'), _metric,
-                                    JOIN("", "[0", _i, ']'), "(S)");
+                                    JOIN("", "XCP_", xcp_idx, ": [", (_i < 10 ? "0" : ""),
+                                         _i, ']'));
                     }
                     else
                     {
                         return JOIN(" ", "GPU", JOIN("", '[', _dev_id, ']'), _metric,
-                                    JOIN("", '[', _i, ']'), "(S)");
+                                    JOIN("", "[", (_i < 10 ? "0" : ""), _i, ']'));
```

Comment:
```suggestion
                                    JOIN("", "[", (_i < 10 ? "0" : ""), _i, ']'), "(S)");
```
Did you remove the "(S)" on purpose? We have added this postfix to show that it is a sampled value.

Reply:
No its an oversight. I will add this.